xml/System.Windows.Xps.Packaging/IXpsFixedDocumentReader.xml

```diff
@@ -1,353 +1,355 @@
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="pageSource" Type="System.Uri" />
+      </Parameters>
+      <Docs>
+        <param name="pageSource">The URI of the page.</param>
+        <summary>Gets a reader for the <see cref="T:System.Windows.Documents.FixedPage" /> with the specified uniform resource identifier (URI).</summary>
+        <returns>An <see cref="T:System.Windows.Xps.Packaging.IXpsFixedPageReader" /> for the page.</returns>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="PrintTicket">
+      <MemberSignature Language="C#" Value="public System.Printing.PrintTicket PrintTicket { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance class System.Printing.PrintTicket PrintTicket" />
+      <MemberSignature Language="DocId" Value="P:System.Windows.Xps.Packaging.IXpsFixedDocumentReader.PrintTicket" />
+      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property PrintTicket As PrintTicket" />
+      <MemberSignature Language="F#" Value="member this.PrintTicket : System.Printing.PrintTicket" Usage="System.Windows.Xps.Packaging.IXpsFixedDocumentReader.PrintTicket" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Printing::PrintTicket ^ PrintTicket { System::Printing::PrintTicket ^ get(); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>ReachFramework</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the <see cref="T:System.Printing.PrintTicket" />, if one exists, that is associated with the <see cref="T:System.Windows.Documents.FixedDocument" />.</summary>
+        <value>A <see cref="T:System.Printing.PrintTicket" /> that contains the default printing options for the document; or <see langword="null" /> if no print ticket exists for the document.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ For more information about the <xref:System.Printing.PrintTicket> part of an XML Paper Specification (XPS) document, see chapter 2 in the XPS specification, which you can obtain at [XPS: Specification and License Downloads](https://www.microsoft.com/whdc/xps/downloads.mspx).  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="N:System.Printing" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Property is not a valid PrintTicket instance.</exception>

```

Comment:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

Reply:
I think we should keep the name of the class instead of translating to a friendly name.   Reactive if you disagree.  Thanks.